### ISSUE TITLE
Installation fails on debian using pip 3 install

### ISSUE THREAD

**Issue by picaso** [ets-labs/python-dependency-injector#158]
```
Downloading dependency-injector-3.4.4.tar.gz (213kB)
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-86gop9jr/dependency-injector/setup.py", line 14, in <module>
        description = readme_file.read()
      File "/usr/lib/python3.4/encodings/ascii.py", line 26, in decode
        return codecs.ascii_decode(input, self.errors)[0]
    UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 13259: ordinal not in range(128)
```

**Comment by rmk135** [comment]
Hello @picaso,

Thanks for reporting this. This needs to be fixed.

**Comment by rmk135** [comment]
Ok, so it's fixed in version 3.4.5. Version 3.4.5 is already been published on PyPI.

Thank you for your contribution,
Roman

PS: I close this issues cause I feel sure that it's fixed, but, please, feel free to reopen it or just comment in case of any problems :)
